yw_website/yw_rest_services/tests.py

```diff
@@ -14,3 +16,24 @@ def test_yw_ping(self):
         bad_route = '/save/pig/'
         bad_response = self.client.get(path=bad_route)
         self.assertNotEqual(bad_response.status_code, 200, msg="Invalid route failed")
+
+    def test_save_upload(self):
+        self.user = User.objects.create_user(username='testuser', password='12345')
```

Comment:
do these tests work if run more than once? usernames are supposed to be unique in our database, so you should generate a unique username each time. Look at [this link](https://docs.python.org/3/library/uuid.html) for reference on how to do this

Reply:
Even if a new test database is created every every time test is run, if there are other tests that are created later where somebody else also creates a user with username `testuser`, we could get poor or inconsistent test results